app/controllers/languages_controller.rb

```diff
@@ -0,0 +1,66 @@
+class LanguagesController < ApplicationController
+  before_action :set_language, only: %i[edit update add_to_volunteer remove_from_volunteer]
+
+  def new
+    authorize Language
+    @language = Language.new
+  end
+
+  def edit
+    authorize @language
+  end
+
+  def create
+    authorize Language
+    @language = Language.new(language_params)
+    @language.casa_org = current_organization
+    respond_to do |format|
+      if @language.save
```

Comment:
since these are both html, this would be neater as 
```
respond_to do |format|
 format.html do
   if @language.save
     redirect_to edit_casa_org_path(current_organization.id), notice: "Language was successfully created." }
   else
     render :new, status: :unprocessable_entity
```

Reply:
that's right! I'll update the next PR, thanks!